api_types.ts

```diff
@@ -996,14 +996,23 @@ export interface ExecutionContext {
 
   /**
    * If this invocation is a part of a crawling execution, like in Coda Brain, then this ID will be provided
-   * to all invocations. That includes invocations of sync `execute` and `executeGetPermissions`, as well as
-   * dynamic table features like `listDynamicUrls`, `getSchema`, and `getName`.
+   * to all invocations as incremental execution id. That includes invocations of sync `execute` 
+   * and `executeGetPermissions`, as well as dynamic table features like `listDynamicUrls`,
+   * `getSchema`, and `getName`.
    *
    * TODO(patrick): Unhide this
    * @hidden
    */
   readonly executionId?: string;
 
+  /**
+   * Provided by a crawling execution, like in Coda Brain, this ID will be the full execution id.
```

Comment:
per @jonathan-codaio 's request, "crawls" are only a certain type of ingestion. Perhaps "full sync" would work here?

Reply:
(and can you correct my bad comment above?)